components/aws/sagemaker/tests/integration_tests/component_tests/test_hpo_component.py

```diff
@@ -58,8 +58,7 @@ def test_hyperparameter_tuning(
 
     # Verify HPO job was successful on SageMaker
     hpo_job_name = utils.read_from_file_in_tar(
-        output_files["sagemaker-hyperparameter-tuning"]["hpo_job_name"],
-        "hpo_job_name.txt",
+        output_files["sagemaker-hyperparameter-tuning"]["hpo_job_name"]
```

Comment:
We are assuming the default value for all the tests, we should also add a test for a custom path.

Reply:
Integration tests won't be able to see a difference for any of these. It only makes a difference when inside the container, then KFP/Argo takes the output and stores it in minio in the same location regardless of the path.

Reply:
Got it, I misunderstood that part